internal/services/stock_exchange/initialise.go

```diff
@@ -0,0 +1,207 @@
+package stock_exchange
+
+import (
+	"Tradeasy/config"
+	model "Tradeasy/internal/model/stock_exchange"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type Response struct {
+	Results []Values `json:"results"`
+}
+type Values struct {
+	Ticker string `json:"ticker"`
+}
+type StockFeed struct {
+	Open      float64 `json:"open"`
+	PrevClose float64 `json:"close"`
+	Ticker    string  `json:"symbol"`
+	LTP       float64 `json:"afterHours"`
+	High      float64 `json:"high"`
+	Low       float64 `json:"low"`
+}
+
+func RandomizerAlgo() {
+
+	var allStocks []model.Stocks
+	config.DB.Model(&allStocks)
+	orderType := []string{"Limit", "Market"}
+	for _, stock := range allStocks {
+
+		//placing buy order
+		orderID := uuid.New().String()
+		rand.Seed(time.Now().UnixNano())
+		idx := rand.Intn(2)
+		order := orderType[idx]
+		min := stock.LTP - int(float64(stock.LTP)*0.01)
+		max := stock.LTP + int(float64(stock.LTP)*0.01)
+		buyOrderBody := OrderRequest{
+			OrderID:         orderID,
+			StockName:       stock.StockName,
+			OrderPlacedTime: time.Time{},
+			OrderType:       order,
+			LimitPrice:      rand.Intn(max-min+1) + min,
+			Quantity:        rand.Intn(100) + 1,
+		}
+		_, err := BuyOrder(buyOrderBody)
+		if err != nil {
+			fmt.Println(err.Error())
+			return
+		}
+
+		//placing sell order
+		orderID = uuid.New().String()
+		rand.Seed(time.Now().UnixNano())
+		idx = rand.Intn(2)
+		order = orderType[idx]
+		min = stock.LTP - int(float64(stock.LTP)*0.01)
+		max = stock.LTP + int(float64(stock.LTP)*0.01)
+		time.Sleep(1 * time.Second)
+		sellOrderBody := OrderRequest{
+			OrderID:         orderID,
+			StockName:       stock.StockName,
+			OrderPlacedTime: time.Time{},
+			OrderType:       order,
+			LimitPrice:      rand.Intn(max-min+1) + min,
+			Quantity:        rand.Intn(100) + 1,
+		}
+		_, err = SellOrder(sellOrderBody)
+		if err != nil {
+			fmt.Println(err.Error())
+			return
+		}
+	}
+	// sleep and run again
+	time.Sleep(120 * time.Second)
+	RandomizerAlgo()
```

Comment:
Run a simple infinite for loop instead of recursion.

Reply:
Done